chef/roles/ceph-radosgw.rb

```diff
@@ -1,5 +1,5 @@
 name "ceph-radosgw"
 description "Ceph RADOS Gateway"
 run_list(
-        "recipe[ceph::radosgw]"
+        "recipe[ceph::role_ceph_radosgw]"
```

Comment:
Style/FirstParameterIndentation: Indent the first parameter one step more than the start of the previous line.